labonneboite/web/static/js/toggle-long-text.js

```diff
@@ -0,0 +1,40 @@
+// from https://justmarkup.com/log/2017/01/truncating-and-revealing-text-the-show-more-and-read-more-patterns/
+
+// cut the mustard
+if ('querySelector' in document && 
+	'addEventListener' in window) {
```

Comment:
tabs and not spaces?

THSI IS WAR!

Reply:
bien vu j'ai pas remarqué

venait d'un script copié du web

will convert to space